locale/en/locale.po

```diff
@@ -164,3 +164,37 @@ msgid "plugins.generic.crossref.successMessage"
 msgstr ""
 "The metadata for this item has been deposited with Crossref. "
 "To view further details, see the submission in the <a href=\"https://doi.crossref.org/servlet/submissionAdmin?sf=detail&submissionID={$submissionId}\">Crossref admin panel</a>."
+
+msgid "plugins.generic.crossref.issueId.required"
+msgstr "Issue ID is required."
+
+msgid "plugins.generic.crossref.onlineIssn.required"
+msgstr "Online ISSN is required."
+
+msgid "plugins.generic.crossref.printIssn.required"
+msgstr "Print ISSN is required."
+
+msgid "plugins.generic.crossref.dateSubmitted.required"
+msgstr "Submission date is required."
+
+msgid "plugins.generic.crossref.dateSubmitted.date"
+msgstr "Submission date must be a valid date."
+
+msgid "plugins.generic.crossref.title.required"
+msgstr "Title is required."
+
+msgid "plugins.generic.crossref.title.en.required"
+msgstr "Title in English is required."
+
+msgid "plugins.generic.crossref.journalTitle.required"
+msgstr "Journal title is required."
+
+msgid "plugins.generic.crossref.journalTitle.en.required"
+msgstr "Journal title in English is required."
```

Comment:
Is the journal title required in the primary locale, or in English? The message doesn't match the rule.